client/rest-high-level/src/main/java/org/elasticsearch/client/transform/transforms/TransformCheckpointingInfo.java

```diff
@@ -49,13 +49,13 @@
         a -> {
             long behind = a[2] == null ? 0L : (Long) a[2];
             Instant changesLastDetectedAt = (Instant) a[3];
-            Instant changesLastSearchedAt = (Instant) a[4];
+            Instant lastSearchTime = (Instant) a[4];
```

Comment:
`lastSearchTime` or `lastSearchedTime`?

Please use the same variable name in both `TransformCheckpointingInfo` classes.